flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogFunctionImpl.java

```diff
@@ -69,10 +76,16 @@ public boolean isGeneric() {
 		return false;
 	}
 
+	@Override
+	public LanguageKind getLanguage() {
```

Comment:
Nit: match the method name with the variable name, i.e.,

Reply:
Updated accordingly.